tests/__mocks__/clipboard_item.ts

```diff
@@ -0,0 +1,21 @@
+class mockClipboardItem implements ClipboardItem {
+  readonly types: string[] = [];
+  readonly presentationStyle = "unspecified";
+  private data = {};
+
+  constructor(dict) {
+    for (const key in dict) {
+      this.types.push(key);
+      this.data[key] = dict[key];
+    }
+  }
+
+  getData(): string {
```

Comment:
Does this really exist in a real ClipboardItem ? I cannot find it in the doc, and this doesn't seems to exist in the browser. I think we're supposed to use `getType() ` to get the content of the clipboard item.

Reply:
no , I forgot to remove it